uiowa/uiowa
# Update Admissions contact info on grad.admissions.uiowa.edu

Resolves #6489

# How to test

```
ddev blt ds --site grad.admissions.uiowa.edu
```
- Ensure the update hook runs without issue.
- The update hook will print out a list of node IDs that were updated. Spot check the results using this list.
- Ensure that a new revision was created.
